Makefile

```diff
@@ -152,3 +152,27 @@ runserver: ## Run the Django application locally
 
 .PHONY: dev-init
 dev-init: load-design-system-templates collectstatic makemigrations migrate createsuperuser ## Run the pre-run setup scripts
+
+.PHONY: functional-tests-up
+functional-tests-up:  ## Start the functional tests docker compose dependencies
+	docker compose -f functional_tests/docker-compose.yml up -d
+
+.PHONY: functional-tests-up-dev
+functional-tests-up-dev:  ## Start the functional tests docker compose dependencies and dev app
+	docker compose -f functional_tests/docker-compose-dev-app.yml up -d
+
+.PHONY: functional-tests-down
+functional-tests-down:  ## Stop the functional tests docker compose dependencies
+	docker compose -f functional_tests/docker-compose-dev-app.yml down
+
+.PHONY: functional-tests-run
+functional-tests-run: collectstatic  ## Run the functional tests
+	DJANGO_SETTINGS_MODULE=cms.settings.functional_test poetry run ./manage.py migrate --noinput
```

Comment:
Might not be clear to all, but, this will only work if you comment out conflicting env vars in `.env` when running locally.

Reply:
Realising it should be passed in with the `--settings` option